pkg/tfgen/generate.go

```diff
@@ -1068,7 +1068,13 @@ func (g *Generator) gatherResource(rawname string,
 	// Ensure there weren't any custom fields that were unrecognized.
 	for key := range info.Fields {
 		if _, has := schema.Schema().GetOk(key); !has {
-			g.warn("custom resource schema %s.%s was not present in the Terraform metadata", name, key)
+			msg := fmt.Sprintf("there is a custom mapping on resource '%s' for field '%s', but the field was not found in the Terraform metadata and will be ignored. Remove the mapping.", rawname, key)
```

Comment:
Actually, can we say "please remove the mapping"? or "To fix, remove the mapping"?

Reply:
the first time I read it, I thought it said "removing the mapping" and missed that it was an instruction